src/Bicep.LangServer/Handlers/BicepDeploymentParametersHandler.cs

```diff
@@ -32,14 +31,14 @@ public record BicepDeploymentParameter(string name, string? value, bool isMissin
     /// </summary>
     public class BicepDeploymentParametersHandler : ExecuteTypedResponseCommandHandlerBase<string, string, string, BicepDeploymentParametersResponse>
     {
-        private readonly ICompilationManager compilationManager;
+        IDeploymentFileCompilationCache deploymentFileCompilationCache;
```

Comment:
Missing `private readonly`.

Reply:
Ah thank you. Update here: https://github.com/Azure/bicep/commit/6cc03a95e5f98e08a3eb114a440aa0eec9aeb70a